File: output/CodeGen/avatar/Python/Java_handraft/codeforces_32_B.java
import java.util.*;

public class codeforces_32_B {
    public static void main ( String [ ] args ) {
        String s = new Scanner ( System.in ).nextLine ( );
        System.out.println ( binary(s) ); 
    }
    public static String binary ( String s ) {
        String result = "";
        int i = 0;
        while ( i < s.length ( ) ) {
            if ( s.charAt ( i ) == '.' ) {
                result += "0";
            } else {
                i++;
                if ( s.charAt ( i ) == '.' ) {
                    result += "1";
                } else {
                    result += "2";
                }
            }
            i++;
        }
        return result;
    }
}

File: output/ChatGPT/avatar/Java/Python/codeforces_32_B.py
import java.util.Scanner

public class codeforces_32_B {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        char[] n = in.next().toCharArray();
        boolean s = false;
        
        for (int i = 0; i < n.length; i++) {
            if (n[i] == '.') {
                if (s) {
                    System.out.print(1);
                    s = false;
                } else {
                    System.out.print(0);
                }
            } else {
                if (s) {
                    System.out.print(2);
                    s = false;
                } else {
                    s = true;
                }
            }
        }
        
        in.close();
    }
}